### DISCUSSION TITLE
[Feat]: "general" category in the discussions?

### DISCUSSION THREAD

**Discussion by zaaakher** [davidjerleke/embla-carousel#694] [Ideas] [answered]
### Which variants of Embla Carousel are relevant to this request?

- [ ] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### Feature description

Upon creating [this](https://github.com/davidjerleke/embla-carousel/discussions/693) discussion I had to choose "show and tell" because it doesn't require me to complete the discussion templates you've setup recently (well done on that btw 👍 )

My suggestion here is perhaps we can have 1 or 2 more categories like this 👇 ?

![image](https://github.com/davidjerleke/embla-carousel/assets/46135573/633abbcd-8958-4ceb-900e-7a0e2cb23633)

Which can be used literally as announcements and general (non-feature, non-Q&A, non-show&tell) discussions.

### Additional Context

Additional details here...

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues
- [X] I agree to follow this project's Contributing Guidelines for feature requests

**Comment by davidjerleke** [comment]
@zaaakher thanks for this. I think the general category should be added as you say. That’s a good idea 🙂.

**Reply by davidjerleke** [reply, marked as answer]
@zaaakher done [here](https://github.com/davidjerleke/embla-carousel/discussions/categories). Thank you for the idea 💡!

**Reply by zaaakher** [reply]
Awesome 👍 Thank you 
https://github.com/davidjerleke/embla-carousel/discussions/categories/general
